python/tests/test_pyvw.py

```diff
@@ -210,3 +210,33 @@ def test_example_namespace():
     assert exm.pop_feature()
     exm.push_features(nmid, ['c', 'd'])
     assert exm.num_features_in() == 4
+
+def test_simple_label():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    sl = pyvw.simple_label('c', weight=0.5)
```

Comment:
Passing a string as the first parameter is in fact a bug, this should only be a float

Reply:
Okay. Then this should raise error in that case. I will change that and add a raise error if string is passed.